src/sbm_model.jl

```diff
@@ -1,3 +1,5 @@
+abstract type sbm end
```

Comment:
Should start with a capital. How about `AbstractSBM`? Since SBM exists.

Reply:
This is actually not used, will remove this.